tests/fixture_setup/_unittests.py

```diff
@@ -128,37 +128,72 @@ def _fake_elffile_extract(self, path):
         glibc = elf.NeededLibrary(name="libc.so.6")
         glibc.add_version("GLIBC_2.2.5")
         glibc.add_version("GLIBC_2.26")
-        return (arch, "/lib64/ld-linux-x86-64.so.2", "", {glibc.name: glibc}, False)
+        return (
+            arch,
+            "/lib64/ld-linux-x86-64.so.2",
+            "",
+            {glibc.name: glibc},
+            False,
+            True,
```

Comment:
maybe worth, maybe not, but a case for this last part of the Tuple returning False might be good.

Reply:
There is a test for static binaries where `is_dynamic` is set to False. It's the `fake-elf-static` test in line 163 (harder to see because indentation make it visually smaller than the other test cases).

Reply:
Great.